google/cloud/storage/client.py

```diff
@@ -1278,6 +1348,7 @@ def create_hmac_key(
         project_id=None,
         user_project=None,
         timeout=_DEFAULT_TIMEOUT,
+        retry=DEFAULT_RETRY,
```

Comment:
Not idempotent, must have no retry argument or retry=None default.

Do you have access to our GCS Consolidated Retry Strategy document? If not, we need to get you in on that.

Reply:
a7bd535715c3e22d921920170828dedda163d721